docker-multinode/common.sh

```diff
@@ -38,7 +38,14 @@ kube::multinode::main(){
   LATEST_STABLE_K8S_VERSION=$(curl -sSL "https://storage.googleapis.com/kubernetes-release/release/stable.txt")
   K8S_VERSION=${K8S_VERSION:-${LATEST_STABLE_K8S_VERSION}}
 
-  ETCD_VERSION=${ETCD_VERSION:-"3.0.4"}
+  CURRENT_PLATFORM=$(kube::helpers::host_platform)
+  ARCH=${ARCH:-${CURRENT_PLATFORM##*/}}
+
+  if [[ ${ARCH} != "arm" ]]; then
+    ETCD_VERSION=${ETCD_VERSION:-"3.0.4"}
+  else
+    ETCD_VERSION=${ETCD_VERSION:-"3.0.4"}
+  fi
```

Comment:
I suppose this is mistake in version for ARM but you should switch condition:
```
if [[ ${ARCH} = "arm" ]]; then

else 
``` 
It will make your code much easier to read. This `!=` is misleading